src/main/java/com/microsoft/azure/proton/transport/ws/WebSocket.java

```diff
@@ -1,7 +1,5 @@
-/*
```

Comment:
Do we really need to made change on the track 1 libraries? 
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cognitiveservices/ms-azure-cs-imagesearch/src/main/java/com/microsoft/azure/cognitiveservices/search/imagesearch/BingImageSearchAPI.java